routes/accounts/weekly_summary.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
```

Comment:
Remove the `use strict`, mostly for consistency.  (When we have built tools, this will happen automatically.)